.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
since this is a package, we can remove this file - the package will never be run from an IDE or independently

Reply:
[673cb3](https://github.com/evoleen/fhir_rest_client/pull/1/commits/7673cb3bf786943d3ef7387beb0a801032c4187d)